build.gradle

```diff
@@ -129,15 +130,19 @@ dependencies {
     // Specify the version of Minecraft to use. If this is any group other than 'net.minecraft', it is assumed
     // that the dep is a ForgeGradle 'patcher' dependency, and its patches will be applied.
     // The userdev artifact is a special name and will get all sorts of transformations applied to it.
-    minecraft 'net.minecraftforge:forge:1.19.2-43.2.21'
+    minecraft "net.minecraftforge:forge:${mc_version}-43.2.21"

```

Comment:
What is the point in leaving the forge version?

Reply:
You are putting to much thought into this